data/sql/updates/pending_db_world/rev_1681810720948358500.sql

```diff
@@ -0,0 +1,7 @@
+-- Bosses on the Ogrila quest chain
+-- Slaag + no dodge
+UPDATE `creature_template` SET `mechanic_immune_mask` = `mechanic_immune_mask`|545470038, `flags_extra` = `flags_extra`|8388608 WHERE `entry` = 22199;
+-- Maggoc, Grulloc, Skulloc Soulgrinder
```

Comment:
I'm not too sure about the dodge. Only 2 creatures in AC have it. TC doesn't have it at all

Reply:
Yeah, neither am I. Probably best not to add.